libs/sdk-ui-kit/styles/scss/main.scss

```diff
@@ -14,3 +14,4 @@
 @import "./header";
 @import "./messages";
 @import "./overlay";
+@import "./remove";
```

Comment:
should not be this "reset" imported as first, to allow others to easily override this?

Reply:
Yes, true. The import order, will move it to the top